testexamples/ethereum_block_handler_test.go

```diff
@@ -25,5 +33,6 @@ func TestHandlers(t *testing.T) {
 	runner := testutils.NewEthereumBlockHandlerTestRunner(t, sourceData)
 	defer runner.Close()
 
-	runner.TestHandler(FindBlockHandler, 0)
+	runner.TestHandlerString(FindBlockHandlerString, 0)
```

Comment:
here 0 should be string

Reply:
The 0 is a count and it will be removed in the next PR with a actual checker on the destination db.

Reply:
addressed in https://github.com/Zettablock/zsource/pull/4